backend/internal/service/models/stocks.go

```diff
@@ -32,7 +32,8 @@ type StockPrice struct {
 }
 
 type StockMatch struct {
-	Order      StockTransaction `json:"order" bson:"order"`
-	QuantityTx int              `json:"quantityTx" bson:"quantity_tx"`
-	PriceTx    int              `json:"priceTx" bson:"price_tx"`
+	Order       StockTransaction `json:"order" bson:"order"`
+	QuantityTx  int              `json:"quantityTx" bson:"quantity_tx"`
+	PriceTx     int              `json:"priceTx" bson:"price_tx"`
+	CostTotalTx int              `json:"costTotalTx" bson:"cost_total_tx"`
```

Comment:
json values need to be `snake_case`, not `camelCase`. that's what the TestDoc expects

Reply:
I was just copying the format of the structs above it. Would this mean all the structs in models/stocks.go must be changed?